docker-compose.yml

```diff
@@ -25,8 +25,7 @@ services:
     ports:
       - "5672"
     volumes:
-      - type: tmpfs
-        target: /var/lib/rabbitmq
+      - tmpfs-rabbitmq:/var/lib/rabbitmq:rw
```

Comment:
This is not anymore a `tmpfs` as in "in-RAM filesystem", is it?

Reply:
No because mac system cannot handle more than one tmpfs
According to the documentation, it was no supported at all, but using only one container had no issue ...
But having two containers with tmpfs cause containers kill others tmpfs containers

Reply:
Can we rename it to something else than `tmpfs` then?

Reply:
Sure, `tmp-rabbitmq`?

Reply:
I want a name that said: It's not important to have persistence
I don't want to leave it without name: it's hard to manage

Reply:
Good for me: changed, pushed and merged :)